python/cudf/cudf/_lib/text.pyx

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
+
+import cudf
+
+from cython.operator cimport dereference
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.string cimport string
+from libcpp.utility cimport move
+
+from cudf._lib.column cimport Column
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.io.text cimport (
+    data_chunk_source,
+    make_source,
+    make_source_from_file,
+    multibyte_split,
+)
+
+
+cpdef read_text(object filepaths_or_buffers,
```

Comment:
This can probably be a plain `def` function. We're never going to benefit from calling the pure C API.